lc0_main.go

```diff
@@ -697,6 +700,10 @@ func hideLc0argsFlag() {
 }
 
 func main() {
+	randBytes := make([]byte, 2)
+	rand.Reader.Read(randBytes)
+	// Ignoring errors from above is deliberate, and unsafe - care should be taken on server before relying on the id field.
```

Comment:
Why not check for errors and set randId to -1?

Reply:
Done - I hope it never errors anyway :)